File: TransCoder_tokenized_test_set_functions/java/MINIMIZE_ABSOLUTE_DIFFERENCE_SUM_TWO_SUBSETS.java
static void subsetDifference ( int n ) {
  int s = n * ( n + 1 ) / 2;
  if ( n % 4 == 0 ) {
    System . out . println ( "First subset sum = " + s / 2 );
    System . out . println ( "Second subset sum = " + s / 2 );
    System . out . println ( "Difference = " + 0 );
  }
  else {
    if ( n % 4 == 1 || n % 4 == 2 ) {
      System . out . println ( "First subset sum = " + s / 2 );
      System . out . println ( "Second subset sum = " + ( ( s / 2 ) + 1 ) );
      System . out . println ( "Difference = " + 1 );
    }
    else {
      System . out . println ( "First subset sum = " + s / 2 );
      System . out . println ( "Second subset sum = " + s / 2 );
      System . out . println ( "Difference = " + 0 );
    }
  }
}

File: TransCoder_tokenized_test_set_functions/python/MINIMIZE_ABSOLUTE_DIFFERENCE_SUM_TWO_SUBSETS.py
def subsetDifference ( n ) :
    s = int ( n * ( n + 1 ) / 2 )
    if n % 4 == 0 :
        print ( "First subset sum = ", int ( s / 2 ) )
        print ( "Second subset sum = ", int ( s / 2 ) )
        print ( "Difference = ", 0 )
    else :
        if n % 4 == 1 or n % 4 == 2 :
            print ( "First subset sum = ", int ( s / 2 ) )
            print ( "Second subset sum = ", int ( s / 2 ) + 1 )
            print ( "Difference = ", 1 )
        else :
            print ( "First subset sum = ", int ( s / 2 ) )
            print ( "Second subset sum = ", int ( s / 2 ) )
            print ( "Difference = ", 0 )
    
